### PR TITLE
Adding publication and ethics statement

### LINKED ISSUE #496: Publication ethics and malpractice statement
For JOSS to be indexed in Scopus, we need to draft a publication ethics and malpractice statement and host it on the JOSS site.

Here's an example from an Elsevier journal: https://www.journals.elsevier.com/asia-pacific-journal-of-sports-medicine-arthroscopy-rehabilitation-and-technology/policies-and-guidelines/publication-ethics-and-publication-malpractice-statement and the guidelines from Scopus on what this document should contain: https://www.elsevier.com/__data/assets/word_doc/0018/116082/pems_june15.docx

### PR DESCRIPTION
**Pull request by arfon**
This PR adds a publication and ethics statement to the JOSS site.

Fixes #496

### CODE REVIEW

File: app/views/home/about.html.erb [hunk 1]
```diff
@@ -107,6 +107,85 @@
...
+        </ul>
+
+        <b>Archiving</b>
+        <ul>
+          <li><%= setting(:abbreviation) %> articles, metadata, and reviews are archived with <a href="https://www.portico.org/">Portico</a>.</li>
+        </ul>
+
+        <b>Ownership and management</b>
+        <ul>
+          <li>Information about the ownership and/or management of a journal shall be clearly indicated on the journal’s web site. (http://www.theoj.org)</li>
```
Comment (arfon) [inline app/views/home/about.html.erb]: @danielskatz - do you think what we have under 'Ownership & Governance' on http://www.theoj.org is sufficient here?

**Comment by arfon** [comment]
@danielskatz - I would appreciate a review of this if you get a chance. These changes are live at https://joss.theoj.org/about#ethics

File: app/views/home/about.html.erb (hunk 1 above)
Comment (danielskatz) [reply app/views/home/about.html.erb]: maybe change "Open Journals and each of its journals are volunteer-led" to "Open Journals and each of its journals are community-organized and volunteer-led"

**Comment by danielskatz** [comment]
maybe change "A journal’s website, including the text that it contains, shall demonstrate that care has been taken to ensure high ethical and professional standards. (see https://joss.theoj.org and https://joss.theoj.org/about)" to "The JOSS website (see https://joss.theoj.org and https://joss.theoj.org/about), including the text that it contains, demonstrates that care has been taken to ensure high ethical and professional standards."

**Comment by danielskatz** [comment]
maybe change "JOSS is not a serial publication - it immediately publishes accepted articles." to "JOSS immediately publishes accepted articles; it is not a serial publication."

**Comment by danielskatz** [comment]
I would fix the 2 J in "As far as we know, the Journal name (Journal of Open Source Software (JOSS)) is unique and not one that is easily confused with another journal or that might mislead potential authors and readers about the Journal’s origin or association with other journals." to "As far as we know, the journal name (Journal of Open Source Software (JOSS)) is unique and not one that is easily confused with another journal or that might mislead potential authors and readers about the journal’s origin or association with other journals."

**Comment by danielskatz** [comment]
I also wonder about JOSS Publication Ethics and Malpractice Statement being a subsection under Ethics and the having a further subsection on publication Ethics that repeats the highest level Ethics content

**Comment by arfon** [comment]
Thanks for the review @danielskatz! I've incorporated your changes and deployed them to the live site.

> I also wonder about JOSS Publication Ethics and Malpractice Statement being a subsection under Ethics and the having a further subsection on publication Ethics that repeats the highest level Ethics content.

Yeah, I'm not entirely happy with this either but I proposed we merge this as-is and revisit/refresh at some point in the future. As it currently reads, the top-level ethics section feels like an overview for authors, the lower section feels like some kind of compliance statement.

**Pull request merged**